pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "schemathesis"
-version = "2.5.1"
+version = "2.5.0"
```

Comment:
There is some conflict here. Could you please rebase against the last version in the repo's master branch?